ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+| supply | an integer denoting how many of this NFT exist |
+| image | a URI pointing to an image representation of the NFT |
+| video | a URI pointing to a video representation of the NFT |
+| audio | a URI pointing to a audio representation of the NFT |
+| domain | a [domain name](https://datatracker.ietf.org/doc/html/rfc1035#section-2.3.1) corresponding to a place that owns or renders this NFT | 
+
+These fields should be self-explanatory.
+
+
+### A Full Example
+
+We start with an official SDF NFT: the [Meridian 2021 hat NFT](https://www.litemint.com/items/GDOA2FECYKBVG2RQYG5JQO6EBLCDBWZ7MFBEZY6AIBJ4NDXV3XEVCVBC/MERIDIAN2021). It's issued by [this account](https://horizon.stellar.org/accounts/GDOA2FECYKBVG2RQYG5JQO6EBLCDBWZ7MFBEZY6AIBJ4NDXV3XEVCVBC/) and stores the following entry in its `data` field:
+
+```json
+{
+  "ipfshash": "UW1heWRHRXNoYkw3SE1Ra0Q2RlpQY01lRXQ2bzE1YUM0cDRQODJWeFE3YW1qeQ=="
+}
+```
+
+Upon decoding the value from base64 (since a `ManageDataOp`'s value is just raw bytes), we get an IPFS CID:
+
+    QmaydGEshbL7HMQkD6FZPcMeEt6o15aC4p4P82VxQ7amjy
+
+Looking this up on IPFS [via CloudFlare](https://cloudflare-ipfs.com/ipfs/QmaydGEshbL7HMQkD6FZPcMeEt6o15aC4p4P82VxQ7amjy) gives us the full metadata:
+
+```json
+{
+  "name": "Meridian 2021: Hat",
+  "description": "Virtual swag from Meridian 2021!  Find out more about Meridian at https://meridian.stellar.org",
+  "image": "ipfs://QmTAQ1e5zD4rxEspDgSZNgqXb73LBEj2wpQkDTj1UvCmv6",
+  "url": "https://ipfs.io/ipfs/QmTAQ1e5zD4rxEspDgSZNgqXb73LBEj2wpQkDTj1UvCmv6",
+  "code": "MERIDIAN2021",
+  "issuer": "GDOA2FECYKBVG2RQYG5JQO6EBLCDBWZ7MFBEZY6AIBJ4NDXV3XEVCVBC",
+  "domain": "meridian.litemint.store",
+  "supply": 50,
+  "video": "ipfs://QmeDxfvNpoizLxP4xLNrx1QuykPSqXEyZL8tVceZGudcvs"
+}
+```
+
+We can see that this account [really does](https://horizon.stellar.org/operations/164563600389443587) own and issue this asset, meaning the NFT is genuine.
+
+## Design Rationale
+TODO.
+
+
+## Security Concerns
+While there are no direct security concerns to the network introduced by this SEP, the trust model of NFTs deserves some clarification.
+
+Digital assets are infinitely and perfectly reproduceable by virtue of being digital. Thus, the goal of an NFT should **not** be to represent ownership or possession of a particular digital asset, but rather to represent the existence of a particular _relationship_ with the issuer.
+
+Those processing an NFT should take care to ensure that the `issuer` field matches the account referencing the NFT, and that the `code` is in fact an asset issued by said account. Without these two links, you cannot make any conclusions about the NFT.
+
+There is another trust issue worth nothing: **supply**. This metadata field _should not be trusted_, and should instead be cross-referenced with the account's distribution operations. Unless an account is completely locked, nothing inherently prevents it from ignoring this `supply` field and issuing new tokens.
```

Comment:
Why include `supply` in the metadata if it requires verification from another source? Is it intended to capture the supply at the time of mint?

Reply:
It's included for compatibility with Litemint, see [here](https://cloudflare-ipfs.com/ipfs/QmaydGEshbL7HMQkD6FZPcMeEt6o15aC4p4P82VxQ7amjy). However, I was hesitant to add it, so I'm fine leaving it off of the official spec.

Reply:
It'd simplify the spec to leave it out. Any field included should be very well defined and intentional imo. Services like litemint can always define extra fields or we can add it later.

@FredericRezeau what is supply? Is it required by litemint?

Reply:
I dropped references to `supply` in 87d6a3bb8367d7ff5285772cd9a6f381ff44dc44, but will leave this open if Frederic wants to chime in.